### ISSUE TITLE
Arkivskåp och skattkistor

### ISSUE THREAD

**Issue by evelinabe** [bjorne/git-pa-svenska#61]
...för visst är väl ens `stash` lite mer av en skattkista och ett repo ett arkivskåp (jo då, arkivskåp är väldigt levande, fråga bara alla handläggare och arkivarier runtom i landet).

**Comment by DavidTagt** [comment]
Ja, jag tycker det låter bättre, Att se ens stash som något mer värdefullt likt en skattkisa är oavsett bättre riktning att gå, men jag har problem med verbet. 

Man kan "stasha" något som i "im gonna stash this until monday" eller "my stash has been stashed full of junk" eller "stashing my candy was the best idea I've had".
"Jag kommer skattkistadetta till på måndag" låter ju tokigt. Samma med "Min skattkista har blivit skattkistad full av skräp", "skattkistande mitt godis är en bra ide" eller "Att lägga mitt godis i min skattkista är det bästa beslutet jag tagit".
"Jag har en skattkista från en annan gren som jag behöver knuffa in i den här grenen"?

Att säga "jag ska lägga detta i min skattkista" blir liksom för långt, "stash that and switch branches" blir "lägg det där i skattkistan och byt gren", funkar väl men vi vill hålla det kortare? 

Arkiv har ju inte detta problem, men jag tycker det låter som man låser in grejer i ett dammigt rum och glömmer bort det. 
Men det är ju bara en personlig föreställning, det går ju att ändra på. 
"Arkivera detta tills måndag" är ju en markant förbättring. "Mitt arkiv har akriverats av en stolle, han arkiverade det fullt i skräp!" likaså (stollen i detta fall är ju jag 😇) och jag tycker det är lättare att säga "Ända sedan jag börjat arkivera mitt godis har det blivit så dammigt".

Man ska se sin stash som en skattkista man arkiverar saker i?
